src/roles/vsd-deploy/tasks/main.yml

```diff
@@ -451,6 +462,16 @@
       timeout_seconds: 1200
       test_interval_seconds: 30
 
+  - name: Allow XMPP Connections after 
```

Comment:
I'd suggest a more descriptive task name here - what is this running after?

Reply:
done